build.sbt

```diff
@@ -14,8 +14,11 @@ scalacOptions ++= Seq(
 
 val sparkVersion = "2.1.0"
 
+parallelExecution in Test := false
+
```

Comment:
I had to disable parallel execution of tests (for now) since the two tests relying on SparkContext/StreamingContext were stepping over each other's toes. A better approach may be to move these kinds of tests into a suite that's configured to run its tests sequentially.